lib/cli_interface.rb

```diff
@@ -26,20 +24,34 @@ def set_lang(input)
     @text_file = YAML::load(File.open(file))
   end
 
-  def menu_game
-    write("#{get_from_file('menu_type_games')}")
+  def display_menu_type_game(default_checked_type = 1)
```

Comment:
Yes I tried to create a sort of convention for the names. Do you think it would be better like that?

- display_game_type_menu
- display_board_size_menu
- display_lang_menu